.github/nitpicks.yml

```diff
@@ -20,3 +20,10 @@
   blocking: true
   pathFilter:
     - '+**/*.dll'
+
+- markdown: |
+    Testing - fail if the file doesn't include "always()"
+  pathFilter:
+    - '.github/workflows/example.yml'
+  contentFilter:
+    - '!always()'
```

Comment:
Do you think non-matching content filters should be their own input? It might be a valid use case for one to want to match a string that does start with a `!` 🤔

Reply:
yeah we can do it that way too! I personally felt like these were two sides of the same coin so it made sense to use contentFilter for both

Reply:
I'll run a test to see what '!(!always())' will do as a content filter

Reply:
That actually appears to work. I ran a test with this as the content filter:
```
 contentFilter:
    - '!(!always())'
```

with this as the yaml, it produced a comment:
`if: 'always()'`

with this as the yaml, it did not produce a comment:
`if: '!always()'`